packages/drivers/routerlicious-driver/src/test/mapWithExpiration.spec.ts

```diff
@@ -286,9 +286,10 @@ describe("MapWithExpiration", () => {
 				for (const map of maps) {
 					map.set(1, "one");
 					map.forEach(() => {
-						assert.notEqual(
-							this,
-							thisArg,
+						// In ESM the the 'this' value is 'undefined', but in CJS it is '{__esModule: true}'.
+						// Therefore, we exempt the test value 'undefined' from the below assertion.
+						assert(
+							this !== thisArg || this === undefined,
```

Comment:
@jason-ha - This is my proposed alternate fix for the CJS/ESM discrepancy: just don't test this one specific value in this one specific test.